packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from '../get-novu-integration';
 import { SelectIntegrationCommand } from './select-integration.command';
 import { decryptCredentials } from '../../encryption';
-import { buildIntegrationKey, CachedQuery } from '../../services';
+import { buildIntegrationKey, CachedQuery } from '../../services/cache';
```

Comment:
Forced to implement this. Need to investigate what's the circular dependency causing this.